src/System.Runtime.Extensions/tests/System/MathF.netcoreapp1.1.cs

```diff
@@ -0,0 +1,368 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Globalization;
+using Xunit.Sdk;
+
+namespace System.Tests
+{
+    public static partial class MathFTests
+    {
+        /// <summary>
+        /// Verifies that two <see cref="float"/> values are equal, within the number of decimal
+        /// places given by <paramref name="precision"/>.
+        /// </summary>
+        /// <param name="expected">The expected value</param>
+        /// <param name="actual">The value to be compared against</param>
+        /// <param name="precision">The number of decimal places (valid values: 0-6)</param>
+        /// <exception cref="EqualException">Thrown when the values are not equal</exception>
+        public static void AssertEqual(float expected, float actual, int precision)
+        {
+            // This is essentially equivalent to the Xunit.Assert.Equal(double, double, int) implementation
+            // available here: https://github.com/xunit/xunit/blob/2.1/src/xunit.assert/Asserts/EqualityAsserts.cs#L46
+
+            var expectedRounded = MathF.Round(expected, precision);
```

Comment:
The use of a `MathF` function here raises a flag for me, since this is a core assertion used by the rest of the tests. It would be nice if the `MathF.Round` tests didn't use this, at the very least.